build/docker-exec

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [[ $# -ne 2 ]] ; then
+    echo "build/docker-exec - run a rake command on a specified ruby version"
+    echo "Usage:"
+    echo "  build/docker-exec 2.2.5 spec"
```

Comment:
These lines are instances of https://github.com/backus/rubocop-rspec/pull/205/files#r77936617